sqldelight-compiler/src/main/kotlin/com/squareup/sqldelight/core/lang/IntermediateType.kt

```diff
@@ -104,6 +107,16 @@ internal data class IntermediateType(
     return sqliteType.prepareStatementBinder(columnIndex, value)
   }
 
+  private fun resolveParent(node: PsiElement): PsiElement {
```

Comment:
The implementation of this function can just be PsiTreeUtil.findParentOfType(node, Queryable::class.java) I think, and then that also removes the cast